addons/sale_stock/tests/test_anglosaxon_account.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.sale_stock.tests.test_anglo_saxon_valuation_reconciliation import TestValuationReconciliation
+from odoo.tests import tagged
+
+@tagged('post_install', '-at_install')
+class TestAngloSaxonAccounting(TestValuationReconciliation):
+
+    def test_cogs_should_use_price_from_the_right_company(self):
```

Comment:
Should I move the test to an existing class as I'm not changing any setup? (but setting it somewhere feels wrong as we are testing a different flow)